.github/workflows/build_prod_on_master_commit.yaml

```diff
@@ -1,10 +1,39 @@
 name: build-apps
 
 on:
-  push:
+  pull_request:
```

Comment:
correct back to push and GitHub realises

Reply:
I think yes

Reply:
Fixed.